03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,36 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>The Daily Planet</title>
-        <link rel="stylesheet" href="css/daily-planet-style.css"/>
-    </head>
-    <body>
+
+<head>
+    <title>The Daily Planet</title>
+    <link rel="stylesheet" href="css/daily-planet-style.css" />
+</head>
+
+<body>.
+    <header>
+        <h1>The Daily Planet</h1>
+    </header>
+    <nav>
+        <a href="http://espn.go.com/">Sports</a>
+        <a href="https://weather.com/">Weather</a>
+        <a href="http://www.austinchronicle.com/">Local</a>
+    </nav>
+    <section>
+        <h1>Trump withhold support</h1>
+        <img width="100%" src="http://i2.cdn.turner.com/cnnnext/dam/assets/160114170540-ryan-trump-mccain-split-overlay-tease.jpg">
+        <p><strong>Washington (CNN)-</strong>Donald Trump is declining to endorse a pair of Republican heavyweights -- House Speaker Paul Ryan and former presidential nominee John McCain -- in their primary fights, another sign of the fractures within the
+            party.
+        </p>
+        <p>Ryan and McCain face long-shot challengers in their respective primary races in Wisconsin and Arizona. Yet their party's presidential nominee would not bless them in an interview with The Washington Post on Tuesday.</p>
+        <p>"I like Paul, but these are horrible times for our country. We need very strong leadership. We need very, very strong leadership. And I'm just not quite there yet. I'm not quite there yet," Trump said of Ryan.</p>
+    </section>
+
+    <aside>
+        <img width="100%" src="http://orig00.deviantart.net/9f66/f/2009/213/4/c/long_cat_is_long_by_plastik_rainbows.jpg" </aside>
+        <footer>

```

Comment:
It looks like you may have not closed the "img" tag here, which is causing the footer to show up inside the Aside tag.